spec/spec_helper.rb

```diff
@@ -7,4 +9,6 @@
     mocks.verify_partial_doubles = true
   end
   config.shared_context_metadata_behavior = :apply_to_host_groups
+
+  WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
remove this
you just need to add `gem 'webmock', require: 'webmock/rspec'` within your `Gemfile`